src/c21server/dashboard/dashboard_server.py

```diff
@@ -0,0 +1,22 @@
+from flask import Flask, jsonify, render_template
+from job_stats import get_jobs_stats
+
+app = Flask(__name__)
+
+@app.route('/dashboard', methods=['GET'])
+def index():
+    job_information = get_jobs_stats();
+    return render_template(
+        'index.html', jobs_waiting=job_information['num_jobs_waiting'],
+        jobs_in_progress=job_information['num_jobs_in_progress'],
+        jobs_done=job_information['num_jobs_done'],
+        jobs_total=job_information['jobs_total']
+    )
+
+@app.route('/data', methods=['GET'])
+def get_dashboard_data():
+    job_information = get_jobs_stats();
+    return jsonify(job_information), 200
+
+if __name__ == '__main__':
+    app.run(port=5000)
```

Comment:
The port number should be documented in the `dashboard.md`.

Eventually, we will want to server the webpage through https via port 443.  If you have never done this, see https://www.digitalocean.com/community/tutorials/how-to-serve-flask-applications-with-gunicorn-and-nginx-on-ubuntu-18-04.  

Short version:  run your `flask` app on port 5000 through `gunicorn` and then have Nginx forward traffic on port 443 to your app.